### DISCUSSION TITLE
why endpoint search input disable in 8.9

### DISCUSSION THREAD

**Discussion by xu1009** [apache/skywalking#8654] [Q&A] [answered]
<img width="898" alt="image" src="https://user-images.githubusercontent.com/22722907/157642231-831892b7-89a3-4917-ad96-273e1d7189a6.png">

when we update skywalking to 8.9 from 8.3, the endpoint input search disapear in ui, the input is very useful,  it can fuzzy search,

**Comment by wu-sheng** [comment, marked as answer]
Endpoint selection supports the same query, and have much less candidates comparing to segments.
So, it is removed. We received reports about endpoint name based traces query take one minute, which is pointless.

**Reply by xu1009** [reply]
maybe， we can keep selection and input，the user decision to use any,

in some scene, we need search some class request, such as request url with some keyword，it is very useful

when we update to 8.9 from 8.3, our center many people very hope the input endpoint can be used

**Reply by wu-sheng** [reply]
It will not be added back, as it caused too much, sorry.
The decision has been made for a long time, not recently.
This is a trillion size data set, we should NOT take the risk to crash the whole ElasticSearch or other storages.
